pkgpanda/util.py

```diff
@@ -150,6 +151,38 @@ def get_requests_retry_session(max_retries=4, backoff_factor=1, status_forcelist
     return session
 
 
+def stream_remote_file_with_retries(out_filename, url, retries=4):
+    def cleanup():
+        try:
+            os.remove(out_filename)
+        except Exception:
+            pass
+
+    while True:
+        with open(out_filename, "w+b") as f:
```

Comment:
I think you should drop the `+` here since you're not reading, only writing.

Reply:
Also taken from the existing code, but it's probably safe to drop it.